src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -625,10 +624,14 @@ namespace winrt::TerminalApp::implementation
 
         else
         {
+            wchar_t guidWString[40] = { 0 };
+            StringFromGUID2(profileGuid, guidWString, 40);
```

Comment:
`Utils::GuidToString` might be easier here

Reply:
I would change this to please thee, but for some reason now my vs 2019 (preview) won't start "the application cannot start." It literally died after my last build. Trying to figure that out is taking longer than expected.

Reply:
ok done